Codeinwp/otter-blocks
# Accordion enhancements

Closes #787, #1215.


### Summary

Adds the following enhancements to the accordion block:
- Option to let only one item open at a time
- Option to change the arrow position (to the right or left of the title)
- Option to change the tag of the title (from `h1` to `h6` or `div`)
- Border controls
- Title typography
- Padding controls
- Box Shadow controls
- Active Item Styling - includes color control for the title and background of the title and content
- Now the arrow's color depends on the text's color. This is to make the arrow visible even in dark mode
- Toggle control for enabling FAQ schema

### Screenshots 
----

### Test instructions

- By default, the option to let only one item open at a time will be true. But because until now it didn't work like this, for old users that had `Initial Open` to `true` for more than 1 item, the option will default to false.
- Make sure it works fine in nested accordions



---- 

### Checklist before the final review
